lighthouse-core/audits/is-on-https.js

```diff
@@ -18,7 +18,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user *why* HTTPS use *for all resources* is important. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'All sites should be protected with HTTPS, even ones that don\'t handle ' +
       'sensitive data. This includes avoiding [mixed content](https://developers.google.com/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content), ' +
-      'where some resources are loaded over HTTP despite the initial request being served' +
+      'where some resources are loaded over HTTP despite the initial request being served ' +
```

Comment:
*ahem* this is why I think we should disable linting for these few lines and just use one long string.

Reply:
was there a discussion about this before?

We could always write a lint rule to find `\S' +` and ban it 😆

Reply:
our lint settings already allows this string to break `max-len` because it has a url in it :)

Reply:
Yes. No reasonable way to locate it. GitHub comment search is non-existent.

Reply:
jk "word wrap" was a pretty good search term. https://github.com/GoogleChrome/lighthouse/pull/9105#discussion_r307599198

Reply:
> our lint settings already allow this string to break max-len because it has a url in it :)

there's no emoji reaction awesome enough to react to this so here you go  

![image](https://user-images.githubusercontent.com/2301202/93805191-fc3ec280-fc0c-11ea-9e09-ecadf329c3aa.png)

Reply:
> jk "word wrap" was a pretty good search term. #9105 (comment)

heh, oh apparently I agreed with you then too.